src/main/frontend/commands.cljs

```diff
@@ -49,14 +49,14 @@
 (defn embed-page
   []
   (conj
-   [[:editor/input "{{{embed [[]]}}}" {:last-pattern slash
-                                       :backward-pos 5}]]
+   [[:editor/input "{{embed [[]]}}" {:last-pattern slash
```

Comment:
Will reducing a bracket influence the `embed` function in previously written notes?

Reply:
It should be working, both `{{}}` and `{{{}}}` are supported. But the code needs a bit of refactoring so that the macro syntax will be `{{{}}}` for org-mode and `{{}}` for other markups.